packages/vc-export/package.json

```diff
@@ -37,8 +37,7 @@
     "@polkadot/keyring": "7.5.1",
     "@polkadot/types": "6.4.2",
     "@polkadot/util": "7.5.1",
-    "@polkadot/util-crypto": "7.5.1",
-    "ajv": "^6.12.5",
+    "@polkadot/util-crypto": "^7.5.1",
```

Comment:
Can we use the `^` here where the other @polkadot dependencies use an exact version?

Reply:
ah, that's a mistake. good catch